<launch>
    <rosparam command="load" file="$(find scene_setup)/config/scene_params.yaml"/>

    <node name="object_handler" pkg="scene_setup" type="object_handler" output="screen"/>
    
    <include file="$(find manipulator_control)/launch/manipulator.launch">
        <args name="sim" value="true"/>
    </include>

</launch>